src/load/azext_load/__init__.py

```diff
@@ -13,13 +13,12 @@ class LoadCommandsLoader(AzCommandsLoader):
 
     def __init__(self, cli_ctx=None):
         from azure.cli.core.commands import CliCommandType
-        custom_command_type = CliCommandType(
-            operations_tmpl='azext_load.custom#{}')
-        super().__init__(cli_ctx=cli_ctx,
-                         custom_command_type=custom_command_type)
+        super().__init__(cli_ctx=cli_ctx)
```

Comment:
Any reason for removing the `custom_command_type` ?

Reply:
custom_commands_type should take in a valid operations_tmpl. Due to refactoring, this is invalid path, and is not required